packages/next-swc/crates/next-core/js/src/entry/app-renderer.tsx

```diff
@@ -203,19 +199,44 @@ async function runOperation(renderData: RenderData) {
       },
     }
   }
+  const availableModules = new Set()
+  const toPath = (chunk: ChunkData) =>
+    typeof chunk === 'string' ? chunk : chunk.path
+  const filterAvailable = (chunk: ChunkData) => {
```

Comment:
This function should be commented to explain what it does. Better yet, we should have this as a utility in Turbopack.